docs/source/federation-version-support.mdx

```diff
@@ -27,7 +27,15 @@ The table below shows which version of federation each router release is compile
     <tbody>
     <tr>
         <td>
-            v1.15.0 and later (<a href="https://github.com/apollographql/router/releases">see latest releases</a>)
+            v1.16.0 and later (<a href="https://github.com/apollographql/router/releases">see latest releases</a>)
```

Comment:
Not sure: but should this say v1.16.0-alpha.0 and later?

Reply:
`1.16.0-alpha.0` is a bit of a magic brownie because it's just an alpha.  It actually _doesn't_ have Federation 2.4.2 in it.  In fact, that version probably _could_ become another alpha (or we would have had to have decided to never ship a 1.16.0.

So, this should actually stay as is.